.evergreen/start-atlas-cloud-cluster.sh

```diff
@@ -17,23 +68,43 @@ function atlascli() {
 }
 
 cleanup() {
-  echo "Scheduling Atlas deployment \`$ATLAS_CLOUD_TEST_CLUSTER_NAME\` for deletion..."
-  atlascli clusters delete $ATLAS_CLOUD_TEST_CLUSTER_NAME --force
+  # Assuming that we want to preserve the cluster if the name was provided
+  # outside of script. Helpful when trying to run the tests locally, you can
+  # automatically create a cluster with a custom name for the first time, but
+  # then re-use it when running the tests again. Don't forget to clean it up
+  # after you're done!
+  if [ -z "$_ATLAS_CLOUD_TEST_CLUSTER_NAME" ]; then
+    echo "Scheduling Atlas deployment \`$ATLAS_CLUSTER_NAME\` for deletion..."
+    atlascli clusters delete $ATLAS_CLUSTER_NAME --force
+  else
+    echo "Custom cluster name provided ($_ATLAS_CLOUD_TEST_CLUSTER_NAME), skipping cluster cleanup"
+  fi
 }
 
 trap cleanup EXIT
 
-echo "Creating Atlas deployment \`$ATLAS_CLOUD_TEST_CLUSTER_NAME\` to test against..."
-atlascli clusters create $ATLAS_CLOUD_TEST_CLUSTER_NAME \
+echo "Creating Atlas deployment \`$ATLAS_CLUSTER_NAME\` to test against..."
+atlascli clusters create $ATLAS_CLUSTER_NAME \
   --provider AWS \
   --region US_EAST_1 \
   --tier M10
 
 echo "Waiting for the deployment to be provisioned..."
-atlascli clusters watch "$ATLAS_CLOUD_TEST_CLUSTER_NAME"
+atlascli clusters watch $ATLAS_CLUSTER_NAME
 
 echo "Getting connection string for provisioned cluster..."
-ATLAS_CLOUD_TEST_CLUSTER_CONNECTION_STRING_JSON="$(atlascli clusters connectionStrings describe $ATLAS_CLOUD_TEST_CLUSTER_NAME -o json)"
+CONNECTION_STRINGS_JSON="$(atlascli clusters connectionStrings describe $ATLAS_CLUSTER_NAME -o json)"
+
+export COMPASS_E2e_ATLAS_CLOUD_SANDBOX_CLOUD_CONFIG=$(
```

Comment:
```suggestion
export COMPASS_E2E_ATLAS_CLOUD_SANDBOX_CLOUD_CONFIG=$(
```

Reply:
As we are passing this to `test-web-sandbox-atlas-cloud` evergreen function (as `qa`), should we check if that's set?

Reply:
Or maybe remove it from there as we are now using `MCLI_OPS_MANAGER_URL` to determine this?

Reply:
Yeah, can drop it from evergreen config, forgot about that, makes sense 👍

Reply:
Done in e284976f3aa1516ebf4b8c8a75d27ba8530b9a9b with the typo fix, thanks for spotting!